errbot/config-template.py

```diff
@@ -93,6 +93,12 @@
 # For example CORE_PLUGINS = ('ACLs', 'Backup', 'Help') you get those names from the .plug files Name entry.
 # For absolutely no plug: CORE_PLUGINS = ()
 
+# Defines an order in which the plugins are getting their callbacks. Useful if you want to have plugins do
+# pre- or post-processing on messages.
+# The 'None' tuple entry represents all the plugins that aren't to be expicitely ordered. For example, if
```

Comment:
Hm. I like the plugin approach because it means I can use the same thing for both (say) Slack and HipChat. Also, it means that it still looks very much like a normal plugin which just adds commands, which is very easy to approach and maintain even not knowing ErrBot's insides.